index.js

```diff
@@ -0,0 +1,11 @@
+/* Get the number n (n>0) to return the reversed sequence from n to 1.
+
+Example : n=5 >> [5,4,3,2,1]*/
+
+const reverseSeq = n => {
+    let table = [];
+    for (i=n; i>0; i--) {
```

Comment:
dla czytelności dobrze by było dodać kilka spacji:
```suggestion
    for (i = n; i > 0; i--) {
```

Reply:
Czy potrafiłbyś to napisać używając `while` ? 😉